[ZING-JVM] IOBalancerStressTest.testEachConnectionUseDifferentSelectorEventually

```
java.lang.AssertionError: activeHandlerEvent count should be at least 1000, but was:50
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.nio.tcp.nonblocking.iobalancer.IOBalancerStressTest.assertBalanced(IOBalancerStressTest.java:174)
	at com.hazelcast.nio.tcp.nonblocking.iobalancer.IOBalancerStressTest.assertBalanced(IOBalancerStressTest.java:97)
	at com.hazelcast.nio.tcp.nonblocking.iobalancer.IOBalancerStressTest.testEachConnectionUseDifferentSelectorEventually(IOBalancerStressTest.java:83)
```

https://hazelcast-l337.ci.cloudbees.com/view/Zing/job/Hazelcast-3.x-ZingJDK7/34/testReport/junit/com.hazelcast.nio.tcp.nonblocking.iobalancer/IOBalancerStressTest/testEachConnectionUseDifferentSelectorEventually/

```
java.lang.AssertionError: activeHandlerEvent count should be at least 1000, but was:35
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.nio.tcp.nonblocking.iobalancer.IOBalancerStressTest.assertBalanced(IOBalancerStressTest.java:172)
	at com.hazelcast.nio.tcp.nonblocking.iobalancer.IOBalancerStressTest.assertBalanced(IOBalancerStressTest.java:95)
	at com.hazelcast.nio.tcp.nonblocking.iobalancer.IOBalancerStressTest.testEachConnectionUseDifferentSelectorEventually(IOBalancerStressTest.java:81)
```

https://hazelcast-l337.ci.cloudbees.com/view/Zing/job/Hazelcast-3.x-ZingJDK6/17/testReport/junit/com.hazelcast.nio.tcp.nonblocking.iobalancer/IOBalancerStressTest/testEachConnectionUseDifferentSelectorEventually/

--- a/hazelcast/src/test/java/com/hazelcast/nio/tcp/nonblocking/iobalancer/IOBalancerStressTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/nio/tcp/nonblocking/iobalancer/IOBalancerStressTest.java
@@ -74,7 +74,7 @@ public class IOBalancerStressTest extends HazelcastTestSupport {
         IMap<Integer, Integer> map = instance1.getMap(randomMapName());
 
         long deadLine = System.currentTimeMillis() + TEST_DURATION_SECONDS * 1000;
-        for (int i = 0; System.currentTimeMillis() < deadLine; i++) {
+        for (int i = 0; System.currentTimeMillis() < deadLine && map.size() < 1000; i++) {
             map.put(i % 1000, i);
         }
 

